src/app/util/mock/attribute-storage.cpp

```diff
@@ -28,12 +28,12 @@
  *    might be fixed with a mock endpoint-config.h
  */
 
-#include <app-common/zap-generated/att-storage.h>
 #include <app-common/zap-generated/attribute-type.h>
 #include <app-common/zap-generated/ids/Attributes.h>
 #include <app/MessageDef/AttributeDataIB.h>
 #include <app/MessageDef/AttributeReportIB.h>
 #include <app/MessageDef/AttributeStatusIB.h>
+#include <app/att-storage.h>
```

Comment:
I know you kept the name, but "att-storage.h" is not really helpful naming, especially since the constants in there are not really about "attribute storage" per se.....  We should figure out what the right name for this is, or whether these constants should go in some existing header.  Followup is ok for this, probably.

Reply:
created #26690